src/tabs/CopyGc/index.tsx

```diff
@@ -0,0 +1,454 @@
+              labelValues={state.flips}
+              labelRenderer={this.renderLabel}
+            />
+          </div>
+        </div>
+        <div>
+          <div>
+            <h3>{state.toSpace === 0 ? 'To Space' : 'From Space'}</h3>
+            <div>
+              {toMemoryMatrix && toMemoryMatrix.length > 0
+                ? toMemoryMatrix.map((item, row) => (
+                    <div style={{ display: 'flex', flexDirection: 'row' }}>
+                      <span style={{ width: 30 }}>
+                        {
+                          // eslint-disable-next-line react/destructuring-assignment
+                          row * this.state.column
+                        }
+                      </span>
+                      {item && item.length > 0
+                        ? // eslint-disable-next-line @typescript-eslint/no-unused-vars, arrow-body-style
+                          item.map((content, column) => {
+                            const color = this.getMemoryColor(content);
+                            const bgColor = this.getBackgroundColor(content);
+                            return (
+                              // eslint-disable-next-line jsx-a11y/mouse-events-have-key-events
+                              <div
+                                style={{ width: 14, backgroundColor: bgColor }}
+                                // onMouseOver={() => this.changeContent(content)}
+                              >
+                                <canvas
+                                  width={10}
+                                  height={10}
+                                  style={{
+                                    backgroundColor: color,
+                                    // margin: 2,
+                                  }}
+                                  // onClick={() => this.updateChild(content)}
+                                />
+                              </div>
+                            );
+                          })
+                        : false}
+                    </div>
+                  ))
+                : false}
+            </div>
+          </div>
+          <div>
+            <h3>{state.toSpace > 0 ? 'To Space' : 'From Space'}</h3>
+            <div>
+              {fromMemoryMatrix && fromMemoryMatrix.length > 0
+                ? fromMemoryMatrix.map((item, row) => (
+                    <div style={{ display: 'flex', flexDirection: 'row' }}>
+                      <span style={{ width: 30 }}>
+                        {
+                          // eslint-disable-next-line react/destructuring-assignment
+                          row * this.state.column + state.memorySize / 2
+                        }
+                      </span>
+                      {item && item.length > 0
+                        ? // eslint-disable-next-line @typescript-eslint/no-unused-vars, arrow-body-style
+                          item.map((content, column) => {

```

Comment:
Dont need `// eslint-disable-next-line @typescript-eslint/no-unused-vars, arrow-body-style` if you remove the column parameter from the lambda function.